app/assets/javascripts/discourse/app/routes/user-activity-bookmarks.js

```diff
@@ -38,6 +38,7 @@ export default DiscourseRoute.extend({
 
         const model = { bookmarks, loadMoreUrl };
         this.session.set("bookmarksModel", model);
+        controller.set("loading", false);
```

Comment:
Should we be setting this in a `finally` hook instead so we don't end up stuck with the loading spinner on error?

Reply:
Done ;)